roles/custom/matrix-media-repo/templates/media-repo/systemd/matrix-media-repo.service.j2

```diff
@@ -0,0 +1,52 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix media-repo
+{% for service in matrix_media_repo_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_media_repo_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Environment="HOME={{ devture_systemd_docker_base_systemd_unit_home_path }}"
+ExecStartPre=-{{ devture_systemd_docker_base_host_command_sh }} -c '{{ devture_systemd_docker_base_host_command_docker }} kill matrix-media-repo 2>/dev/null || true'
+ExecStartPre=-{{ devture_systemd_docker_base_host_command_sh }} -c '{{ devture_systemd_docker_base_host_command_docker }} rm matrix-media-repo 2>/dev/null || true'
+
+ExecStartPre={{ devture_systemd_docker_base_host_command_docker }} create \
+			--rm \
+			--name=matrix-media-repo \
+			--log-driver=none \
+			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
+			--cap-drop=ALL \
+			--network={{ matrix_docker_network }} \
+			-p {{ matrix_media_repo_port }}:{{ matrix_media_repo_port }} \
+			{% if matrix_media_repo_metrics_enabled %}
+			-p {{ matrix_media_repo_metrics_port }}:{{ matrix_media_repo_metrics_port }} \
+			{% endif %}
+			-v {{ matrix_media_repo_base_path }}:/data:z \
```

Comment:
It'd be better if we use `--mount` instead of `-v`. We probably don't need the magical `z` thing.

The better thing aobut using `--mount` is that `-v` is magical as well. If the target directory does not exist for some reason and you use `-v`, it will automatically create it (and likely with some `root:root` ownership).

In cases where the directory does not exist, something is probably wrong, so it's better to have the service fail and let you investigate instead of automatically creating directories.

------

We should probably introduce a new `matrix_media_repo_data_path` variable and use it for this data.

```yaml
matrix_media_repo_data_path: "{{ matrix_media_repo_base_path }}/data"
```

This frees up the base directory for being used for other things in the future (environment variables file, Traefik labels, other configuration, some migration files). Passing the whole "base path" as `/data` means that any other files/directories we create within it will be directly accessible by matrix-media-repo.

This is a bit of a breaking change though.